Solutions/Azure Purview/Analytic Rules/AzurePurviewSensitiveDataDiscovered.yaml

```diff
@@ -0,0 +1,38 @@
+id: 8001949e-8039-4e75-b9d2-7d591bcc7026
```

Comment:
Can you please provide a new Id for this analytic rule as this ID is already used for another analytic rule?